test/features/conditionalLogic.feature

```diff
@@ -58,4 +58,12 @@ Feature: Set of scenarios testing show & hide functionality and conditional logi
     When I do NOT meet the condition for showing the field in the tab
     Then the 'Text Field 3' field will NOT be visible on the 'Conditional Tab 1' tab
 
-
+  @bugfix @RDM-4622
+  Scenario: Page Show Condition works for read only field in previous event
+    Given a case type containing conditionals exists
```

Comment:
not sure if it's me, but I don't find this scenario clear enough. I've asked Aman and he can't understand it as well. I think we should improve it

Reply:
have broken it into 2 smaller scenarios